exporters/pelorus/timeutil.py

```diff
@@ -101,3 +104,16 @@ def to_iso(dt: datetime) -> str:
         )
 
     return dt.astimezone(timezone.utc).strftime(_ISO_ZULU_FMT)
+
+
+def is_outofdate(timestring: str) -> bool:
+    """
+    Helper function, which allows to filter out metrics which are older then
+    the accepted time. This is to ensure Prometheus will not try to scrape too
+    old metrics.
+    """
+    if to_epoch_from_string(timestring) < datetime.now() - timedelta(
```

Comment:
I think this is wrong

should not be
```python
datetime.now() - to_epoch_from_string(timestring) < timedelta(minutes=METRIC_TIMESTAMP_THRESHOLD_MINUTES)
```

and this if already returns a boolean, just do
```python
return condition
```

Reply:
Add a unit test to this function

Reply:
No the:

```
datetime.now() - to_epoch_from_string(timestring)
```

Represents the time difference between now and the time of measurement

If the above is greater then timedelta() then it should be outofdate (return True).

So we could use datetime.now() on the left side, but then the logic would be:


```
datetime.now() - to_epoch_from_string(timestring) > timedelta(minutes=METRIC_TIMESTAMP_THRESHOLD_MINUTES)
```